code/modules/clothing/suits/ego_gear/realized.dm

```diff
@@ -297,6 +297,34 @@
 	armor = list(RED_DAMAGE = 90, WHITE_DAMAGE = 40, BLACK_DAMAGE = 90, PALE_DAMAGE = 40)
 	realized_ability = /obj/effect/proc_holder/ability/aimed/yin_laser
 
+/obj/item/clothing/suit/armor/ego_gear/realization/repentance
+	name = "repentance"
+	desc = "If you pray hard enough, perhaps god will answer it?"
+	icon_state = "repentance"
+	armor = list(RED_DAMAGE = 90, WHITE_DAMAGE = 40, BLACK_DAMAGE = 40, PALE_DAMAGE = 90)
+	realized_ability = /obj/effect/proc_holder/ability/prayer
+
+/obj/item/clothing/suit/armor/ego_gear/realization/nest
+	name = "living nest"
+	desc = "Grow eternally, let our nest reach the horizon!"
+	icon_state = "nest"
+	armor = list(RED_DAMAGE = 90, WHITE_DAMAGE = 60, BLACK_DAMAGE = 60, PALE_DAMAGE = 50)
+	realized_ability = /obj/effect/proc_holder/ability/nest
+
+/obj/item/clothing/suit/armor/ego_gear/realization/nest/equipped(mob/user, slot, initial = FALSE)
+	. = ..()
+	if(slot == ITEM_SLOT_OCLOTHING)
+		addtimer(CALLBACK(src, .proc/Spawn,user), 10 SECONDS)
+
+/obj/item/clothing/suit/armor/ego_gear/realization/nest/dropped(mob/user)
+	return ..()
+
+/obj/item/clothing/suit/armor/ego_gear/realization/nest/proc/Reset(mob/user)
+	src.Spawn(user)
+
+/obj/item/clothing/suit/armor/ego_gear/realization/nest/proc/Spawn(mob/user)
+	addtimer(CALLBACK(src, .proc/Reset,user), 10 SECONDS)
+	new/mob/living/simple_animal/hostile/naked_nest_serpent_friend(get_turf(user))
```

Comment:
This will infinitely spawn the worms at the equippers location, forever. There's no unequip check, no variable that tracks the timer so it can be stopped, nothing. It can even be stacked.